scripts/cloudlab/start_onenode_vhive_cluster.sh

```diff
@@ -26,34 +26,31 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="$( cd $DIR && cd .. && cd .. && pwd)"
 SCRIPTS=$ROOT/scripts
 
+echo Clean up host resources if left after previous runs
 $SCRIPTS/github_runner/clean_cri_runner.sh
 
-CTRDLOGDIR=$1
-
-if [ -z $CTRDLOGDIR ] && [ -z "$GITHUB_RUN_ID" ] ; then
-    echo "[ERROR]: The log-directory argument is missing"
-    echo "USAGE: start_onenode_vhive_cluster.sh <Directory to store logs>" 
-    exit -1
-else
-    if [ -z "$CTRDLOGDIR" ]; then
-        CTRDLOGDIR=/tmp/ctrd-logs/$GITHUB_RUN_ID
-    fi
-fi
-
-echo "using $CTRDLOGDIR for storing logs"
-
+CTRDLOGDIR=/tmp/ctrd-logs/$GITHUB_RUN_ID
```

Comment:
If custom folder location is to be removed, it would be a good idea to `echo` the folder location.

Reply:
I moved that output to the bottom